### DISCUSSION TITLE
Why use tostring-implementation instead of Lombok?

### DISCUSSION THREAD

**Discussion by reynadess** [trigunam/tostring-implementation#14] [General]
Hi @trigunam 
I went through source code, it is well documented. Well written. I was thinking to solve one of your open [issues](https://github.com/trigunam/tostring-implementation/issues/12). 
After researching, `toString` annotation of [lombok](https://projectlombok.org/features/ToString) does all of that.
I want to understand what differently are we trying to do here.
Thank you
Sai Chaitanya Raparti

**Comment by trigunam** [comment]
Thanks @reynadess This project goes way before Lombok. 
I think everyone is using the Lombok package it works really well, I have personally tried it in my current project.

Thanks for your message @reynadess really appreciate you taking a look at it. I will update the readme to use lombok instead.
